packages/shared/ReactTypes.js

```diff
@@ -128,3 +126,47 @@ export type ReactEventTarget = {|
   displayName?: string,
   type: Symbol | number,
 |};
+
+type AnyNativeEvent = Event | KeyboardEvent | MouseEvent | Touch;
+
+export type ReactResponderEvent = {
+  nativeEvent: AnyNativeEvent,
+  target: Element | Document,
```

Comment:
Are these global flow types? Will flow pass with these in fbsource/React Native?

Reply:
These are globals for Flow. We can revise them when we come to the RN work to add support for these events, but this should pass CI now.